docs/topics/exceptions.rst

```diff
@@ -36,7 +36,7 @@ CloseSpider
 For example::
 
     def parse_page(self, response):
-        if 'Bandwidth exceeded' in response.body:
+        if b'Bandwidth exceeded' in response.body:
```

Comment:
Working with bytes could be difficult, I think it's better if we just use `str` :

```suggestion
        if 'Bandwidth exceeded' in response.text:
```

Reply:
> @Gallaecio why won't it work in Python 2?

I had no idea that the `b''` syntax was supported in Python 2. It indeed works.

Reply:
I’m not against this suggestion, but I don’t see why working with bytes would be a problem here.

Reply:
It becomes an issue when you are working with non-English websites that uses different encoding.

I believe docs should set good example and use strings when looking for text string without trusting the underlying binary representation.